### PR TITLE
feat(new-wallet): talisman

### PR DESCRIPTION
**Pull request by smartdev914**
- Add talisman wallet.
- Update the substrate tools box so that can support talisman, polkadot.js and keystore wallet.

### CODE REVIEW

File: packages/swapkit/helpers/src/modules/assetValue.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export class AssetValue extends BigIntArithmetics {
     return this.chain === chain && this.symbol === symbol;
   }
 
-  eq(assetValue: AssetValue) {
+  eq(assetValue: AssetValue): boolean {
```
Comment (towanTG) [inline packages/swapkit/helpers/src/modules/assetValue.ts:87]: Let TS infer type here

File: packages/swapkit/helpers/src/modules/swapKitError.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ const errorCodes = {
   core_wallet_keplr_not_installed: 10107,
   core_wallet_okx_not_installed: 10108,
   core_wallet_keepkey_not_installed: 10109,
+  core_wallet_talisman_not_installed: 10110,
+  core_wallet_not_keypair_wallet: 10111,
```
Comment (towanTG) [inline packages/swapkit/helpers/src/modules/swapKitError.ts:26]: What wallet is this referring to?

**Comment by towanTG** [comment]
@smartdev914 could you open PRs from the SK repo itself?

Currently you are still using your private fork.

Will make it easier for me to check some stuff in code.

File: packages/swapkit/helpers/src/modules/assetValue.ts (hunk 1 above)
Comment (anthon-dev) [reply packages/swapkit/helpers/src/modules/assetValue.ts:87]: faced a type error cause of return type missing.

File: packages/swapkit/helpers/src/modules/swapKitError.ts (hunk 2 above)
Comment (anthon-dev) [reply packages/swapkit/helpers/src/modules/swapKitError.ts:26]: This error is used to identify the keypair wallet.
The talisman and polkadot.js is not a keypair wallet.

**Comment by anthon-dev** [comment]
> could you open PRs fro
For only this repo, I updated some an existing a branch of my personal repo.
In the future, will work on sk repo itself.

**Pull request merged**